### PR TITLE
Feature: Able to customize crudactions and speeddial buttons globally

### PR DESCRIPTION
**Pull request by martn001**

### CODE REVIEW

File: src/VuetifyResource.vue [hunk 1]
```diff
@@ -327,6 +352,40 @@ export default {
...
+            delete: drop.color || 'red',
+            create: create.color || 'indigo',
+            all: basic ? basic.color : null,
+            onlyCrudActions : basic && !basic.onlySpeedDial,
+            onlySpeedDial : basic && !basic.onlyCrudActions,
+          };
+        },
+        crudColors() {
+          return {
+            edit: this.colors.onlyCrudActions ? this.colors.all : this.colors.edit,
```
Comment (jvhellemondt) [inline src/VuetifyResource.vue:369]: All is een standaard ofzo?

**Review by jvhellemondt — APPROVED** [review]

File: src/VuetifyResource.vue (hunk 1 above)
Comment (ricardovanlaarhoven) [reply src/VuetifyResource.vue:369]: Perhaps `default` would be better

**Review by ricardovanlaarhoven — CHANGES_REQUESTED** [review]
I'm missing the documentation update in the readme.md
And this is a new feature so it can't be a new hotfix version.
this should get a new minor version.

**Review by ricardovanlaarhoven — APPROVED** [review]
Approved if this is tested in a real project and this feature works, it doesn't break anything and you can still change vuetify's sass variables.

File: src/VuetifyResource.vue [hunk 2]
```diff
@@ -327,6 +352,41 @@ export default {
...
+            create: this.colors.onlyCrudActions && this.colors.default  ? this.colors.default : this.colors.create,
+          };
+        },
+        speedDialColors() {
+          return {
+            edit: this.colors.onlySpeedDial && this.colors.default ? this.colors.default : this.colors.edit,
+            delete: this.colors.onlySpeedDial && this.colors.default ? this.colors.default : this.colors.delete,
+            create: this.colors.onlySpeedDial && this.colors.default ? this.colors.default : this.colors.create,
+          };
+        },
```
Comment (jvhellemondt) [inline src/VuetifyResource.vue:381]: Goed -- maar veel herhaling.

File: src/VuetifyResource.vue [hunk 3]
```diff
@@ -327,6 +352,41 @@ export default {
...
+        crudIcons() {
+            return {
+                edit: VuetifyResourceOptions.style.edit.icon || this.$vuetify.icons.values.edit,
+                delete: VuetifyResourceOptions.style.drop.icon || this.$vuetify.icons.values.delete,
+                create: VuetifyResourceOptions.style.create.icon || this.$vuetify.icons.values.add,
+            };
+        },
+        crudIconSize: () => VuetifyResourceOptions.style.default ? VuetifyResourceOptions.style.default.size : null,
```
Comment (jvhellemondt) [inline src/VuetifyResource.vue:389]: Inconsistent met voorgaande computed. ` crudIconSize() {VuetifyResourceOptions.style.default ? VuetifyResourceOptions.style.default.size : null}`

**Pull request merged**